### ISSUE TITLE
Error using Mask RCNN: Lambda Layer: Defining Model

### ISSUE THREAD

**Issue by YashRunwal** [matterport/Mask_RCNN#2328]
Hello guys,

I am currently using Matterport Mask RCNN to train on custom dataset.
I have annotated my images using labelme and then made respective changes to the Dataset Class. 
As far as I can tell, the changes are working. 

When I try to run the following command, I get an error:
```

MODEL_DIR = 'logs'

model = modellib.MaskRCNN(mode="training", config=config,
                          model_dir=MODEL_DIR)
```

Error:

```
Lambda layers are not well suited to stateful computation; instead, writing a
subclassed Layer is the recommend way to define layers with
Variables.

```

I am using following versions:
Tensorflow: 2.2.0
Keras: 2.4.3

I would really appreciate your help with this.

Regards,
Yash

**Comment by Jeharul-Hussain** [comment]
Hi Yash,

Try using Tensorflow 1.15
Keras = 2.3

Tensorflow versions  > 2 still do not support few of the libraries which Mask RCNN deals with.

Also make sure that you supply appropriate values to the config

**Comment by nataliameira** [comment]
Hi @YashRunwal ! Were you able to solve this problem?

**Comment by YashRunwal** [comment]
@nataliameira Yes I downgraded the tensorflow version as suggested by @Jeharul-Hussain

**Comment by Hitesh319** [comment]
> @nataliameira Yes I downgraded the tensorflow version as suggested by @Jeharul-Hussain

Could you tell how to downgrade tensorflow version?

**Comment by nataliameira** [comment]
Hey @Hitesh319 
Hey
I trained my model and visualized the data.
My training was held on June 3, 2021.
I used:

```
pip uninstall keras-nightly
pip uninstall -y tensorflow
h5py==2.10.0
tensorflow==1.15
tensorflow-gpu==1.15.0
keras==2.1
```
